networkx/algorithms/centrality/tests/test_current_flow_betweenness_centrality.py

```diff
@@ -134,6 +134,11 @@ def test_solvers(self):
             for n in sorted(G):
                 np.testing.assert_allclose(b[n], b_answer[n], atol=epsilon)
 
+    def test_lower_kmax(self):
+        G = nx.complete_graph(4)
+        with pytest.raises(nx.NetworkXError, match="Increase kmax or epsilon"):
+            nx.approximate_current_flow_betweenness_centrality(G, kmax=4)
+
```

Comment:
Looking at the implementation of `approximate_current_flow_betweenness_centrality`, it doesn't seem that the `kmax` parameter is actually used... should it perhaps be removed instead? Or perhaps there's a bug in the implementation where the iteration is actually supposed to be over `kmax` instead of `k`?

Reply:
Let's go ahead and move this discussion elsewhere - the test looks good for the current behavior so let's go ahead and add it for now. Thanks for looking into this @chimaobi-okite , it's certainly and interesting find!